frame/glutton/README.md

```diff
@@ -0,0 +1,5 @@
+# Glutton Pallet
+
+Pallet that consumes `ref_time` and `proof_size` of a block. Based on the
+`Compute` and `Storage` parameters the pallet consumes the adequate amount
+of weight. 
```

Comment:
It'd be nice to have a note here with a warning that this pallet should never be used on a chain with actual value, as well as more about why it exists and what for it should be used (instead of only what it's doing).

Reply:
I think there was one originally, but then I removed it since I thought we deploy it directly to the relay as well. Will re-add.